src/models/fields/IllPerson.ts

```diff
@@ -0,0 +1,7 @@
+export enum IllPerson {
```

Comment:
Same here, it should not be added by this PR.

Reply:
Removed as requested.